doc/format.rst

```diff
@@ -755,18 +755,25 @@ monospaced font; in reST, *double* backticks are for ``monospaced font``,
 whereas the behavior of single backticks is defined by the default role. This
 leads to the following style recommendations:
 
-- Module, class, function, method, and attribute names should render as
-  hyperlinks in monospaced font (e.g. :any:`numpy`); depending on project
-  settings, this may be accomplished simply be enclosing them in single
-  backticks. If the hyperlink does not render as intended, explicitly
-  include the appropriate role and/or namespace.
+- For example, module, class, function, method, and attribute names
+  should be enclosed in single backticks. On a case-by-case basis,
+  instances may also be included in single backticks, like :py:obj:`None`,
+  :py:obj:`True`, :py:obj:`False`.
```

Comment:
@Carreau this renders like:
<img width="925" alt="image" src="https://github.com/user-attachments/assets/59097496-130c-4885-919a-69f15d7c668e" />

Seeing only the rendered version and not the underlying reST, I would assume from the text that these object names only need to be included in single backticks to render like this, but that's not quite right. Should these go at the end of the paragraph, after the idea of explicitly including the role and/or namespace is mentioned?